packages-ts/gauntlet-terra-contracts/src/commands/abstract/executionWrapper.ts

```diff
@@ -3,14 +3,21 @@ import { Result } from '@chainlink/gauntlet-core'
 import { TerraCommand, TransactionResponse } from '@chainlink/gauntlet-terra'
 import { AccAddress } from '@terra-money/terra.js'
 import { logger, prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { Query } from './inspectionWrapper'
 
 export type BeforeExecutionContext<Input, ContractInput> = {
```

Comment:
Is there any other type of context? Why don't we just call it `ExecutionContext`?